io.elementary.calendar.yml

```diff
@@ -71,14 +71,13 @@ modules:
       - '-DENABLE_WEATHER=OFF'
       - '-DWITH_OPENLDAP=OFF'
       - '-DENABLE_INSTALLED_TESTS=OFF'
+      - '-DENABLE_TESTS=OFF'
       - '-DENABLE_GTK_DOC=OFF'
       - '-DENABLE_EXAMPLES=OFF'
     sources:
       - type: archive
-        url: https://download.gnome.org/sources/evolution-data-server/3.46/evolution-data-server-3.46.4.tar.xz
-        sha256: a59b254145c59facd7c7b5382db78dc5f0ed1f6a6e9b8fe7d5e75959f93c0cc8
-      - type: patch
-        path: 'eds-gdatasession.patch'
```

Comment:
Shouldn't we delete this path too since it's no longer used anywhere?

```
[ryo@b760m ~/work/tmp/calendar (david/bump-eds =)]$ git ls-files "*eds-gdatasession.patch*"
eds-gdatasession.patch
[ryo@b760m ~/work/tmp/calendar (david/bump-eds =)]$ git grep -n eds-gdatasession.patch
[ryo@b760m ~/work/tmp/calendar (david/bump-eds =)]$
```

Reply:
Oops, yes, fixed that, thank you!